functional/keylime_create_policy-static-data/main.fmf

```diff
@@ -0,0 +1,18 @@
+summary: Tests the keylime_create_policy tool
+description: |
+    Provide different types of input to the keylime_create_policy tool and
+    ensure that it produces the expected runtime policy.
+contact: Stefa Berger <stefanb@linux.ibm.com>
+component:
+  - keylime
+test: ./test.sh
+framework: beakerlib
+tag:
+  - CI-Tier-1
+require:
+  - jq
+recommend:
+  - keylime
+duration: 1m
+enabled: true
+extra-nitrate: TC#0613885
```

Comment:
Please remove this line. This ID is for RH internal purposes and is unique for every test.

Reply:
Fixed.